Userland/Libraries/LibJS/Runtime/VM.cpp

```diff
@@ -703,7 +706,7 @@ void VM::enqueue_promise_job(NativeFunction& job)
 void VM::run_queued_finalization_registry_cleanup_jobs()
 {
     while (!m_finalization_registry_cleanup_jobs.is_empty()) {
-        auto* registry = m_finalization_registry_cleanup_jobs.take_first();
+        auto registry = m_finalization_registry_cleanup_jobs.take_first();
```

Comment:
Leftover change from earlier patch iteration?

Reply:
gah! i need to stop hurrying so much, fixed! :)